coffee_core/src/coffee.rs

```diff
@@ -355,7 +356,7 @@ impl PluginManager for CoffeeManager {
         let status = repository.upgrade(&self.config.plugins).await?;
         for plugins in status.plugins_effected.iter() {
             self.remove(plugins).await?;
-            self.install(plugins, verbose, false).await?;
+            self.install(plugins, verbose, false, None).await?;
```

Comment:
uh you are not using the branch in reality :/ so this is only the step one to support the command, then you need to modify the `self.install` too